doc/sphinx/Makefile

```diff
@@ -6,6 +6,9 @@ include Makefile.sphinx
 CHPL2RST     = ./run-in-venv.bash ./chpl2rst.py
 CHPL2RSTOPTS = --output=rst --prefix=source/primers/primers --link=master
 
+CHPLDOC_HELLO_DIR = source/examples/hellos
```

Comment:
Non-show stopper: `chpldoc` is not part of this process, making the `CHPLDOC` prefix somewhat confusing imo